src/ucp/core/ucp_request.inl

```diff
@@ -151,7 +151,7 @@ static UCS_F_ALWAYS_INLINE void
 ucp_request_complete_tag_recv(ucp_request_t *req, ucs_status_t status)
 {
     ucs_trace_req("completing receive request %p (%p) "UCP_REQUEST_FLAGS_FMT
-                  " stag 0x%"PRIx64" len %zu, %s",
+                  " stag 0x%" PRIx64 " len %zu, %s",
```

Comment:
why needed to add spaces around PRIx64 ?

Reply:
For fixing this error.

https://github.com/openucx/ucx/pull/5552#issuecomment-671231079

```
  CXX      common/gtest-test_helpers.o
  CXX      common/gtest-test_obj_size.o
In file included from /scrap/azure/agent-01/AZP_WORKSPACE/2/s/contrib/../src/ucp/rma/rma.h:11,
                 from /scrap/azure/agent-01/AZP_WORKSPACE/2/s/contrib/../src/ucp/core/ucp_request.h:22,
                 from /scrap/azure/agent-01/AZP_WORKSPACE/2/s/contrib/../test/gtest/common/test_obj_size.cc:16:
/scrap/azure/agent-01/AZP_WORKSPACE/2/s/contrib/../src/ucp/core/ucp_rkey.h:97:23: error: invalid suffix on literal; C++11 requires a space between literal and string macro [-Werror=literal-suffix]
                       "(remote md_map 0x%"PRIx64")", \
                       ^
cc1plus: all warnings being treated as errors
make[3]: *** [common/gtest-test_obj_size.o] Error 1
make[3]: *** Waiting for unfinished jobs....
make[3]: Leaving directory `/scrap/azure/agent-01/AZP_WORKSPACE/2/s/build-test/test/gtest'
```